bertopic/representation/_llamacpp.py

```diff
@@ -93,14 +114,15 @@ def __init__(
         self,
         model: Union[str, Llama],
         prompt: str = None,
+        system_prompt: str = None,
         pipeline_kwargs: Mapping[str, Any] = {},
         nr_docs: int = 4,
         diversity: float = None,
-        doc_length: int = None,
+        doc_length: int = 100,
```

Comment:
Let's keep the value as it was, otherwise we have to do this for all instances of `doc_length` across all models.